### PR TITLE
LTP and exposure updates

### PR DESCRIPTION
**Pull request by stevenyyan**

### CODE REVIEW

File: context/OMEContext.tsx [hunk 1]
```diff
@@ -167,10 +167,9 @@ export const OMEStore: React.FC<Children> = ({ children }: Children) => {
                         maxAsk: maxAsk,
                     },
                 });
-                // precision here isnt that necessary to fixing it to 20 is ok
-                const ltp = new BigNumber(
-                    Web3.utils.fromWei((res?.LTP ?? 0).toLocaleString('fullwide', { useGrouping: false })),
-                );
+                // Precision here isn't that necessary
+                // Fixing it to 20 is ok
```
Comment (dospore) [inline context/OMEContext.tsx]:
```suggestion
```

**Review by dospore — COMMENTED** [review]
Is this the only place ltp is used??

**Comment by stevenyyan** [comment]
> Is this the only place ltp is used??

looks like it, didn't find anywhere else it's used

**Comment by dospore** [comment]
> > Is this the only place ltp is used??
> 
> looks like it, didn't find anywhere else it's used

yeah looks like it

**Review by dospore — APPROVED** [review]

**Pull request merged**